openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Alerts.spec.js

```diff
@@ -0,0 +1,258 @@
+      .should('be.visible')
+      .click();
+    cy.get('.ant-modal-header')
+      .should('be.visible')
+      .should('contain', `Delete ${alertForAllAssets}`);
+    cy.get('[data-testid="confirmation-text-input"]')
+      .should('be.visible')
+      .type(DELETE_TERM);
+    interceptURL('DELETE', 'api/v1/alerts/*', 'deleteAlert');
+    cy.get('[data-testid="confirm-button"]')
+      .should('be.visible')
+      .should('not.disabled')
+      .click();
+    verifyResponseStatusCode('@deleteAlert', 200);
+
+    toastNotification('Alert deleted successfully!');
+    cy.get('table').should('not.contain', alertForAllAssets);
+  });
+
+  it('Create new alert for all data assets and multiple filters', () => {
+    interceptURL('GET', '/api/v1/alerts/*', 'createAlert');
+    //Click on create alert button
+    cy.get('button').contains('Create alert').should('be.visible').click();
+    verifyResponseStatusCode('@createAlert', 200);
+    //Enter alert name
+    cy.get('#name').should('be.visible').type(alertForAllAssets);
+    //Enter description
+    cy.get('#description').should('be.visible').type(description);
+    //Click on all data assets
+    cy.get('[title="All Data Assets"]').should('be.visible').click();
+    cy.get('[title="All Data Assets"]').eq(1).click();
+    //Select filters
+    cy.get('button').contains('Add Filters').should('exist').click();
+    cy.get('#filteringRules_0_name').invoke('show').click();
+    //Select first owner
+    cy.get('[title="Owner"]').should('be.visible').click();
+    cy.get('.ant-select-selection-overflow')
+      .should('be.visible')
+      .click()
+      .type('Engineering');
+    cy.get('[title="Engineering"]').should('be.visible').click();
+    cy.get('#name').should('be.visible').click();
+
+    //Select second owner
+    cy.get('.ant-select-selection-overflow')
+      .should('be.visible')
+      .click()
+      .type('Applications');
+    cy.get('[title="Applications"]').should('be.visible').click();
+    cy.get('#name').should('be.visible').click();
+
+    //Select include/exclude
+    cy.get('[title="Include"]').should('be.visible').click();
+    cy.get('[title="Include"]').eq(1).click();
+
+    //Select Destination
+    cy.get('button').contains('Add Destination').should('exist').click();
+    cy.get('.ant-select-selection-placeholder')
+      .contains('Select Source')
+      .click({ force: true });
+    cy.wait(1000);
+    cy.get('.ant-select-item-option-content').contains('Email').click();
+    cy.wait(500);
+    //Enter email
+    cy.get(
+      '.ant-form-item-control-input-content > .ant-select > .ant-select-selector > .ant-select-selection-overflow'
+    )
+      .eq(1)
+      .click()
+      .type('testuser@openmetadata.org');
+    //Click save
+    cy.get('[type="submit"]').contains('Save').click();
+    toastNotification('Alerts created successfully.');
+    cy.get('table').should('contain', ALERTS.name);
+  });
+
+  it.only('Create new alert for Test case data asset', () => {
```

Comment:
We should remove this to run all the tests right

Reply:
Yes my bad